sqjobs/tests/djsqjobs_tests-fixture.json

```diff
@@ -0,0 +1,47 @@
+[
+    {
+        "fields": {
+            "name": "Adder1",
+            "task": "adder",
+            "kwargs": "{\"num1\":1, \"num2\":2}",
+            "schedule": "1 * * * *",
+            "timezone": "Europe/Madrid",
+            "created_on": "2015-07-26T14:11:10Z",
+            "next_execution": "2015-07-26T15:01:00Z",
+            "enabled": true
+        },
+        "model": "djsqjobs.periodicjob",
+        "pk": 2
+
+    },
+    {
+        "fields": {
+            "name": "Divider1",
+            "task": "divider",
+            "kwargs": "{\"num1\":2, \"num2\":1}",
+            "schedule": "10 * * * *",
+            "timezone": "Europe/Madrid",
+            "created_on": "2015-07-26T14:11:10Z",
+            "next_execution": "2015-07-26T15:10:00Z",
+            "enabled": true
+        },
+        "model": "djsqjobs.periodicjob",
+        "pk": 3
+
+    },
+    {
+        "fields": {
+            "name": "FutureDivider",
+            "task": "divider",
+            "kwargs": "{\"num1\":2, \"num2\":1}",
+            "schedule": "10 * * * *",
+            "timezone": "Europe/Madrid",
```

Comment:
Looks like all your timezones in your fixture are `Europe/Madrid`, using a different one with a big offset like Japan would make this more robust and might spot a bug.